### DISCUSSION TITLE
Should we write a blog post about fud?

### DISCUSSION THREAD

**Discussion by sampsyo** [cucapra/futil#461] [Ideas]
I think it was @rachitnigam's suggestion that we try writing a blog post about how cool fud is, even independently of Calyx itself. Seems like fun, and I thought it might be interesting to discuss what would go into it—@sgpthomas, since it's your baby, does this sound fun to you?

Just some ideas about what to cover:

- The original inspiration: the `clang` driver : the Clang C-to-LLVM compiler :: fud :: Calyx
- Observing how crappy this usually is in the conventional HLS world (e.g., Vivado HLS's Tcl scripts)
- The nifty generic pipeline-assembly design strategy

**Comment by rachitnigam** [comment]
We wrote one! https://calyxir.org/post/one-year/
